.drone.env

```diff
@@ -1,4 +1,4 @@
 # The test runner source for API tests
-APITESTS_COMMITID=ad4b3fc55c2f9ef8e2dfa6eabb095ac0ddea9e17
-APITESTS_BRANCH=master
+APITESTS_COMMITID=8c6b221aeecc307cf97a206970cb79d42d2ac2c6
+APITESTS_BRANCH=fix-reva
```

Comment:
ocis PR is merged. please adjust the branch and commit

Reply:
@S-Panta ^